copilot/data-store/manifest.yml

```diff
@@ -36,20 +36,26 @@ variables:                    # Pass environment variables as key value pairs.
   FLASK_ENV: ${COPILOT_ENVIRONMENT_NAME}
   # Sentry DSN is OK to be public see: https://docs.sentry.io/product/sentry-basics/dsn-explainer/#dsn-utilization
   SENTRY_DSN: https://6a2623e302e641ba88eabe6675a70ddf@o1432034.ingest.sentry.io/4505390859747328
-#secrets:                      # Pass secrets from AWS Systems Manager (SSM) Parameter Store.
+secrets:                      # Pass secrets from AWS Systems Manager (SSM) Parameter Store.
+    NOTIFY_FIND_API_KEY: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/NOTIFY_FIND_API_KEY
+    NOTIFY_FIND_REPORT_DOWNLOAD_TEMPLATE_ID: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/NOTIFY_FIND_REPORT_DOWNLOAD_TEMPLATE_ID
```

Comment:
I've just noticed this is a secret value - it doesn't need to be, we can have it in the `variables` block above

Reply:
Removed NOTIFY_FIND_REPORT_DOWNLOAD_TEMPLATE_ID from here and added the to DefaultConfig with real value.